PLGMAG2V2-84: Add FAQ for deprecated to new migration

Vinod-MultiSafepay opened the pull request:
### Basic
- We only accept documentation that is proven to work on our LIVE API
- At least one approval is needed before pull requests can be merged
- All content is written in US English
- The terminology used must be in accordance with our naming conventions

### Text style
- Paths are formatted like: _Configuration → System → Plugin → MultiSafepay_
- Use single quotation marks in the front matter of markdown files (e.g., title: 'Title of the article')
- Use _italics_ when describing GUI elements (e.g., buttons, links and paths)
- When a sentence ends with an email address, don't use a period (e.g. "[...] contact us at <example@example.com>")

### Images
- Screenshot recommended sizes: Width: 820px - Height: auto

### Aliases
- When renaming or deleting a page, add an alias of the deprecated URL to the page users should be redirected to. This prevents external links from returning 404 errors. Check the [Hugo Documentation](https://gohugo.io/content-management/urls/#aliases) for more information.

Stanislau-MultiSafepay commented on content/integrations/ecommerce-integrations/magento2/faq/migrating-to-new-plugin.md:
```diff
@@ -0,0 +1,57 @@
+---
+title : "What should i do when migrating to plugin version 2?"
```
I suppose 'i' should be capital

Kris-MultiSafepay replied:
;)

Stanislau-MultiSafepay commented on content/integrations/ecommerce-integrations/magento2/faq/migrating-to-new-plugin.md:
```diff
@@ -0,0 +1,57 @@
+### 4. Changes in the checkout
+For the following payment methods, the payment flow has been changed from [Redirect to Direct](/faq/api/difference-between-direct-and-redirect/). This applies to the following payment methods: Afterpay, Request to Pay, Direct Debit, Einvoicing, in3 and Pay After Delivery (Betaal na Ontvangst).
+
+For these payment methods, we have included extra fields in the checkout. This means that if you are using a customised checkout, you now not only have to account for the iDEAL issuers checkout field, but also other checkout fields for the other payment methods mentioned above.
```
I would add here some attention mark, cause it's important

Vinod-MultiSafepay replied:
hmm i made the text bold, i don't know a better way to make the text stand out more. I want to avoid exclamation marks. Any suggestions?

Stanislau-MultiSafepay commented on content/integrations/ecommerce-integrations/magento2/faq/migrating-to-new-plugin.md:
```diff
@@ -0,0 +1,57 @@
+
+For these payment methods, we have included extra fields in the checkout. This means that if you are using a customised checkout, you now not only have to account for the iDEAL issuers checkout field, but also other checkout fields for the other payment methods mentioned above.
+
+#### Example of the Afterpay checkout in the deprecated plugin
```
maybe rename it on "Differences between the Afterpay checkout in the deprecated plugin and in new one" or something like this? and merge topic below in this one ?

Stanislau-MultiSafepay commented on content/integrations/ecommerce-integrations/magento2/faq/migrating-to-new-plugin.md:
```diff
@@ -0,0 +1,57 @@
+---
+Version 2 of our plugin is a different plugin and has been built from the ground up. This also means that there are some differences, compared to the deprecated plugin. This article will describe in detail all of the most important differences, to prevent you running in to unexpected behaviour.
+
+### 1. Installation of the new plugin
```
Maybe we have some small guide on how to delete the old plugin? Will be great if we put links on some article: "How to delete?" and on installation guide for new one

Kris-MultiSafepay reviewed (changes requested):
Please note that a page with the same purpose [already exists](https://docs.multisafepay.com/integrations/ecommerce-integrations/magento2/faq/how-can-i-update-the-plugin-for-magento2/). Please delete this FAQ article and add alias to the new article.

Kris-MultiSafepay commented:
I think we are building an informational maze around the Magento 2 plugin(s). Currently we refer to the new and old plugin as:
| New       | Old        |
|-----------|------------|
| New       | Deprecated |  
| Version 2 | Version 1  |

Vinod-MultiSafepay commented:
> I think we are building an informational maze around the Magento 2 plugin(s). Currently we refer to the new and old plugin as:
> 
> New	Old
> New	Deprecated
> Version 2	Version 1
> I don't disagree with naming it _version 1_ and _version 2_. However, given there's already a 'Version 1' and 'Version 2' of Magento (aka _Magento 1_ and _Magento 2_), the new version of our Magento 2 plugin would be named _Magento 2 plugin version 2_. @Thomas-MultiSafepay perhaps you have a clarifying insight into the intuitive naming of this plugin that can be consistently applied?

I removed 'old' everywhere from the text and changed it to 'deprecated' for consistency. Please check if everything is all right now.
